src/Script.sol

```diff
@@ -11,4 +11,50 @@ abstract contract Script {
         address(bytes20(uint160(uint256(keccak256('hevm cheat code')))));
 
     Vm public constant vm = Vm(VM_ADDRESS);
+
+    // Calcuate the corresponding contract creation address for a given address and nonce
+    function addressFrom(address origin, uint256 nonce) internal pure returns (address creation) {
```

Comment:
Thoughts on renaming to something like `computeCreateAddress`, that way adding another `computeCreate2Address` fits nicely?